webapp/components/_new/generic/UserAvatar/UserAvatar.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div :class="['user-avatar', size && `--${this.size}`]">
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-08T15:50:27Z" title="Wednesday, January 8th 2020, 4:50:27 pm +01:00">Jan 8, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,106 @@
+<template>
+  <div :class="[`size-${this.size}`, 'user-avatar']">
```


I am still trying to figure out the best way to add utility classes – like here for the size. In the [button migration PR](https://github.com/Human-Connection/Human-Connection/blob/be30b55f651a834687dc47c8a28befab24d6e793/webapp/components/_new/generic/BaseButton/BaseButton.vue) I add the class names in a computed property and I think it's more readable than this but I'm not yet sure it will work for all our components and am open to suggestions! :grin: